tests/transports-integrations/tests/utils/config_loader.py

```
@@ -0,0 +1,309 @@
+            config_path = project_root / "config.yml"
+
+        self.config_path = Path(config_path)
+        self._config = None
+        self._load_config()
+
+    def _load_config(self):
+        """Load configuration from YAML file"""
+        if not self.config_path.exists():
+            raise FileNotFoundError(f"Configuration file not found: {self.config_path}")
+
+        with open(self.config_path, "r") as f:
```

Ruff (0.11.9)

Unnecessary mode argument

Remove mode argument

(UP015)